src/App.jsx

```diff
@@ -56,7 +56,15 @@ function App() {
               register={register}
             />
           </div>
-          <Button text="Log in" />
+          <div className="form__buttons">
+            <Button text="Log in" />
+            <br />
+            <Button
+              type="button"
+              styleGoogle="button--google"
```

Comment:
`styleGoogle` is not a conventional prop name. Rename `styleGoogle="button--google"` to `className="button--google"` and ensure your Button component supports className.